rero_ils/modules/items/api.py

```diff
@@ -547,10 +548,33 @@ def last_location_pid(self):
             return loan_location_pid
         return self.location_pid
 
+    def can_extend(self, loan):
+        """Checks if an item can be renewed for the borrowed patron."""
```

Comment:
Is the patron borrowed? :wink: `Checks if the patron as the possibility/has rights/can renew this item.`